precompile/contract_native_minter.go

```diff
@@ -64,6 +70,13 @@ func (c *ContractNativeMinterConfig) Address() common.Address {
 
 // Configure configures [state] with the desired admins based on [c].
 func (c *ContractNativeMinterConfig) Configure(_ ChainConfig, state StateDB, _ BlockContext) {
+	for to, amount := range c.InitialMint {
+		if amount != nil {
+			bigIntAmount := (*big.Int)(amount)
+			state.AddBalance(to, bigIntAmount)
```

Comment:
should we check for overflow?

Reply:
hm nice question, I don't normally see any overflow guard for big ints. what do you think? @aaronbuchwald

Reply:
We should add overflow guards to protect against this since this is up to the user now.

Every place where we do this, it's up to a trusted contract that can mint funds, but we should add overflow checks within the native minter to make sure that nobody minting very aggressively runs into overflow issues.

Reply:
Created an issue here to track: https://github.com/ava-labs/subnet-evm/issues/248. We have the option to either add overflow guards or make sure that it's clearly documented (and that it's a reasonable expectation) to say that the user of the native minter precompile will ensure there is never on the order of enough funds to overflow a uint256.

I think it should actually be reasonable to require that of the user of the native minter precompile.

Reply:
can we add a comment about this decision here?